src/types/types.ts

```diff
@@ -0,0 +1,34 @@
+/** 
+ * Preview: Represents a function to handle previewing with a source string.
+*/
+export type PreviewCallback = { onPreview: (src: string) => void };
+
+/** 
+ * UICSS: Represents a key-value pair of CSS properties.
+*/
+export type UICSS = { [key: string]: string };
+
+/** 
+ * UIStatus: Represents different statuses like EMPTY, UPLOADING, FILLED.
+*/
+export type UIStatus = {EMPTY: string, UPLOADING: string, FILLED: string};
+
+/** 
+ * UIAttributes: Represents a key-value pair of attributes.
+*/
+export type UIAttributes = { [key: string]: any};
+
+/** 
+ * Tunes: Represents a tone with name, icon, title, toggle, and optional action.
```

Comment:
Since it is not tunes, just config for them:
```suggestion
 * User configuration of Image block tunes. Allows to add custom tunes through the config
```